### PR TITLE
Add tine

### PR DESCRIPTION
**Pull request by mzch**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/tine.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Tine
+website_url: https://www.tine-groupware.de/
+description: tine is the ideal software for digital collaboration in companies and organizations. From powerful groupware functionalities to clever add-ons, tine combines everything to make daily team collaboration easier.
```
Comment (Rabenherz112) [inline software/tine.yml:3]:
```suggestion
description: Software for digital collaboration in companies and organizations. From powerful groupware functionalities to clever add-ons, tine combines everything to make daily team collaboration easier.
```

File: software/tine.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
```
Comment (Rabenherz112) [inline software/tine.yml:7]:
```suggestion
  - Docker
```
[Installation Guide](https://tine-docu.s3web.rz1.metaways.net/operators/Installation_Guide/) only mentions Docker

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

File: software/tine.yml (hunk 1 above)
Comment (mzch) [reply software/tine.yml:3]: Ok.

File: software/tine.yml (hunk 2 above)
Comment (mzch) [reply software/tine.yml:7]: Thanks.

**Review by mzch — COMMENTED** [review]
plz commit sugested change.

**Review by nodiscc — APPROVED** [review]

**Review by ryangurn — APPROVED** [review]

**Pull request merged**